src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,14 @@
-export const CommentList = () => <>Put the list here</>;
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) =>
+  comments.length > 0 ? (
+    <div className="CommentList">
+      {comments &&
```

Comment:
This condition is redundant, you already check the length of the comment array above